src/views/UserSettings.vue

```diff
@@ -94,7 +94,34 @@ export default {
         this.languageTranslationCompletion = await localeManager.calculateTranslationCompletion(this.userSettingsForm.selectedLanguage.code)
       }
     },
-
+    'userSettingsForm.selectedCountry': function (newValue, oldValue) {
```

Comment:
i just tested. it works, but it feels kinda strange that when changing the country, both the language & currency are changed without the user realizing :thinking: 
- we should have a short message popup (like when uploading a proof)
- or separate the behavior and not automatically make the changes..

Reply:
Language will not be automatically changed, the current selected language (or browser language or default English) will still be the one selected. But in the dropdown it will show up first, followed by languages of the country and then other languages.
I suppose the same can be done for currency. Or have a popup. But ultimately we will change the currency dropdown to a checkbox list for the user to choose currencies that he is paying with (issue #359 )
I will leave it that way for now and do another PR for currency checkbox dropdown.

Reply:
I tested again and the currency changes. can we have a small `snackbar` to let the user know ?

also the code below seems very complex... I'm not sure we need to go into all these automation, vs just let users choose a country / a language / a currency :thinking:

Reply:
Alright, I will update #372 to take this into consideration and revert to decoupling of currency.